tools/swift-api-digester/ModuleAnalyzerNodes.cpp

```diff
@@ -701,7 +721,16 @@ bool SDKNode::operator==(const SDKNode &Other) const {
       }
       LLVM_FALLTHROUGH;
     }
-    case SDKNodeKind::DeclType:
+    case SDKNodeKind::DeclType: {
```

Comment:
What if I'm adding a conformance in an extension, and the extended type is from a different module? You really should model conformances as top-level entities, and not as part of types. Then adding a new conformance is fine (just like adding a new top-level function), but removing one is not allowed.

Reply:
If the extension is for a type from a different module, the tool will synthesize the type definition in its module dump ([here](https://github.com/apple/swift/blob/master/tools/swift-api-digester/ModuleAnalyzerNodes.cpp#L1189)). So the standalone extension won't be missed.